src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2777,6 +2779,58 @@ static void window_ride_vehicle_update(rct_window* w)
     widget_invalidate(w, WIDX_TAB_2);
 }
 
+static OpenRCT2String window_ride_vehicle_tooltip(
+    rct_window* const w, const rct_widgetindex widgetIndex, rct_string_id fallback)
+{
+    auto ride = get_ride(w->number);
+    if (ride == nullptr)
+        return { STR_NONE, {} };
+
+    switch (widgetIndex)
+    {
+        case WIDX_VEHICLE_TRAINS:
+        case WIDX_VEHICLE_TRAINS_DECREASE:
+        case WIDX_VEHICLE_TRAINS_INCREASE:
+        {
+            auto ft = Formatter();
+            ft.Increment(12);
```

Comment:
What's up with these increments that didn't exit before? There isn't any data prior nor any rewind later, so they seems a bit confusing (to me, at least)

Reply:
The string id associated with this needs the args in those positions. If you look at the chain of adds that this was removed from you will see where the number comes from